packages/core/src/amazonqFeatureDev/controllers/chat/controller.ts

```diff
@@ -172,23 +169,6 @@ export class FeatureDevController {
         const session = await this.sessionStorage.getSession(tabId)
 
         switch (session?.state.phase) {
```

Comment:
We may no longer need the switch case here given that we added more points to allow users to vote feedback.

Reply:
okay